SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security
```

Comment:
Is there a way to have an org wide document rather than per repo?

Reply:
We have the option of putting this in `.github` like Microsoft does [here](https://github.com/microsoft/.github/blob/main/SECURITY.md). It will fill in the "security" tab if a repo doesn't already have one, just confirmed from checking an old Microsoft [example](https://github.com/microsoft/sudokumaster-wp?tab=security-ov-file)

<img width="741" alt="image" src="https://github.com/user-attachments/assets/8d04504c-22bc-4626-b38c-fcd1dcf3bf31" />

I can relocate this draft to `vega/.github` if we prefer!

Reply:
Added a modified proposal to the shared repo  https://github.com/vega/.github/pull/20 . It will fill in for any repos that are missing a local security advisory.